Automattic/woocommerce-payments
# Not focusing email field when showing error message

Fixes #4538

#### Changes proposed in this Pull Request

When an error happens after the user inputs the OTP, the modal closes and a message is display bellow the email field. The email field will get focused so it is possible the autocompletion box covers part of the message.
This PR updates the code not to focus the email field when the modal disappears and an error message is displayed.

#### Testing instructions

1. Force an error on `?wc-ajax=wcpay_init_platform_checkout`.
1. Go to the checkout page.
1. Use an email you have in your browser autocomplete and exists in Woopay.
1. Type the OTP.
1. The modal will disappear, an error message will be display, the field should not be focuesed and a autocompletion box ahould not appear.

-------------------


**Post merge**
